packages/beeq/src/components/drawer/_storybook/bq-drawer.stories.tsx

```diff
@@ -0,0 +1,82 @@
+import type { Args, Meta, StoryObj } from '@storybook/web-components';
+import { html } from 'lit-html';
+
+import mdx from './bq-drawer.mdx';
+import { DRAWER_PLACEMENT } from '../bq-drawer.types';
+
+const meta: Meta = {
+  title: 'Components/Drawer',
+  component: 'bq-drawer',
+  parameters: {
+    docs: {
+      page: mdx,
+    },
+  },
+  argTypes: {
+    open: { control: 'boolean' },
+    placement: { control: 'select', options: [...DRAWER_PLACEMENT] },
+    'disable-close-click-outside': { control: 'boolean' },
+    'disable-close-esc-keydown': { control: 'boolean' },
+    'enable-backdrop': { control: 'boolean' },
+    // Events
+    bqShow: { action: 'bqOpen' },
+    bqHide: { action: 'bqClose' },
```

Comment:
You should map to the custom Events emitted, not to the methods 😉 
```suggestion
    bqOpen: { action: 'bqOpen' },
    bqClose: { action: 'bqClose' },
```